openedx/features/partners/tests/test_login.py

```diff
@@ -0,0 +1,113 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from openedx.features.partners.tests.factories import PartnerUserFactory, PartnerFactory
+from student.tests.factories import UserFactory
+
+
+class ResetPasswordTestCases(APITestCase):
```

Comment:
This class says it's for `ResetPassword`, but the test cases written are for `user_login`. Please rename this class, and also add a docstring if necessary.